docs/tutorials/getting-started.md

```diff
@@ -40,9 +40,56 @@ $ cd redash-ui-tests
 When you're ready dive right into the next steps to learn about how to work
 with the tests.
 
-## Run the test suite
+#### Redash Setup
+
+The redash instance must be setup before running the tests. Start the images ```docker-compose up -d``` in the root of the directory in which the project was downloaded too. This will download the images if you don't have them. It will also start the container in the background. You can view the status of the containers via ```docker-compose ps```. You can also stop the docker container by running ```docker-compose stop```.
+
+All the following commands should also be run within the same directory that you downloaded the project too.
+
+Run these following commands to setup Redash:
+1. Create Database:
+```
+docker-compose run --rm server create_db
+```
+2. Create test Database
+```
+docker-compose run --rm postgres psql -h postgres -U postgres -c "create database tests"
+```
+3. Create a default user named ```rootuser``` with a password ```IAMROOT``` and an organization ```default```.
+```
+docker-compose run --rm server /app/manage.py users create_root root@example.com "rootuser" --password "IAMROOT" --org default
+```
+4. Create a new datastore named ```ui-tests```.
```

Comment:
AFAIK `ds` stands for data source

https://github.com/mozilla/redash/blob/master/redash/cli/__init__.py#L34